src/lib/snack-bar/simple-snack-bar.ts

```diff
@@ -23,7 +24,15 @@ import {MAT_SNACK_BAR_DATA} from './snack-bar-config';
   encapsulation: ViewEncapsulation.None,
   preserveWhitespaces: false,
   changeDetection: ChangeDetectionStrategy.OnPush,
+  animations: [
+    trigger('contentFade', [
+      state('void', style({opacity: '0'})),
+      state('visible', style({opacity: '1'})),
+      transition('void => visible', animate('350ms linear'))
```

Comment:
Should we use the animations from core?

```
`${AnimationCurves.STANDARD_CURVE} ${AnimationDurations.ENTERING}`
```

Reply:
Does that work now with AOT? I thought string interpolation in animation [broke it](https://github.com/angular/material2/blob/master/src/lib/snack-bar/snack-bar-container.ts#L41-L42).

Reply:
That comment is pretty old. It seems to work with AoT now.